tim_mendoza/routes/movies_routes.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var moviesRouter = express.Router();
+var bodyParser = require('body-parser');
+var Movie = require(__dirname + '/../models/movie');
+
+var handleError = function(err, res) {
+  res.send('server error');
```

Comment:
You should use the error information in some way (usually console.log is used). There's no point passing it to your function otherwise.